README.md

```diff
@@ -14,12 +14,24 @@ Requirements
 Should work on any RHEL, Debian, Ubuntu, SUSE, and FreeBSD distributions.
 
 This cookbook depends on Sean O'Meara's [line cookbook](https://github.com/someara/line-cookbook)
+This cookbook depends on Sander van Zoest's [sysctl cookbook](https://github.com/svanzoest-cookbooks/sysctl)
+Both of these cookbooks are available from the Chef supermarket.
 
 ### Attributes
 
 * `nfs['packages']`
   - Case switch in attributes to choose NFS client packages dependent on platform.
 
+* `nfs['client_services']`
+  - Names of the client resources to start and enable.
+
+* `nfs['client_nfsv_services']`
+  - Names of the client resources to start when running nfs v4.
+
+* `nfs['client_not_restartable_services']`
```

Comment:
I do not like this.  Its a terrible variable name.  The way its coded in this PR is inconsistent with the documentation, as there is an unused `client_restartable_services` hardcoded in the attributes.  I think there might be a more elegant solution here, by just separating out portmap as its own recipe, and not having it notify/subscribe to be reloaded.  Since nowhere on any platform, are we configuring portmap; it would make sense to exclude it from service reloading in that sense.  It will keep us from adding even more attributes, which conditionally need set for platform-specific cases.